Gemfile

```diff
@@ -29,6 +29,8 @@ group :development do
   gem 'quiet_assets'
   gem 'better_errors'
   gem 'binding_of_caller'
+  # Help to add/remove n+1 query.
+  gem 'bullet'
```

Comment:
gems are self explanatory, so dont add comments for things easily google'd or common knowledge